stuff/javascript/jasmine-standalone-2.8.0/spec/getPerfectNumbersSpec.js

```diff
@@ -0,0 +1,19 @@
+describe('getPerfectNumbers', function() {
+    it('should return a 0', function() {
```

Comment:
TODO: change the it('should... to more semantic one, like the following:

it('should return a "is a perfect number" for a perfect number input') {